docs/src/examples/components/Box/Performance/ContextPropNesting.perf.tsx

```diff
@@ -0,0 +1,78 @@
+import { ThemeInput } from '@stardust-ui/react'
```

Comment:
Do you want to merge these perf tests to master?

Reply:
Will remove them before merge 👍